swap_meet/item.py

```diff
@@ -1,2 +1,19 @@
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, category = "", condition=0):
+        self.category = category
+        self.condition = condition
+
+    def __str__(self):
+        return "Hello World!"
+
+    def condition_description(self):
```

Comment:
Based on the README, this method "should describe the condition in words based on the value, assuming they all range from 0 to 5."

A lot of our classes default to a 0 condition if an argument isn't given. What should be returned then?